packages/manager/modules/vps/src/upscale/upscale.controller.js

```diff
@@ -180,9 +176,17 @@ export default class UpscaleController {
     );
   }
 
-  getRangeEliteConfigurationPricing() {
+  isStorageSelectionDisabled(storage) {
+    const { cores, memory } = this.rangeConfiguration;
+    const rangeName = this.range.formattedName.toLowerCase();
+    const plans = this.catalog.plans.map((plan) => plan.planCode);
+    this.planCode = `vps-${rangeName}-${cores}-${memory}-${storage}`;
```

Comment:
regarding name of function `isStorageSelectionDisabled` we expect boolean as return.
but in same function you are assigned a value to `this.planCode`.

I guess you have to assign the value somewhere else.

Reply:
@brofik, @mohammed-zahaf's comment is valid. Can you please look into it?